### ISSUE TITLE
System.UnauthorizedAccessException: 

### ISSUE THREAD

**Issue by PAHeartBeat** [firebase/quickstart-unity#92]
Hi Team,

We are facing some weird issue in Firebase Analytics, We have already added latest **Google Mobile Ads** and **Firebase Analytics** with it Play service Resolve, here **Google Mobile Ads** has it's own play service Resolver which works fine, but when we import **Firebase Analytics** It also comes with its own Play service resolver files so we have deleted files came with **Google Mobile Ads**

We have already added **GoogleService-Info.plist* in a root of Assets folder of our unity project. Build works fine, it gives error in log (error don't crash the game) error log is as bellow given

```
System.UnauthorizedAccessException: Access to the path "/private/var/mobile/Containers/Data/Application/6C32F3D5-A815-447C-A4A7-68DB4C907A57/.config" is denied.
  at Firebase.AppOptions.getCPtr (Firebase.AppOptions obj) [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectoriesInternal (System.String path) [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectory (System.String path) [0x00000] in <filename unknown>:0 
  at System.IO.DirectoryInfo.Create () [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectoriesInternal (System.String path) [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectory (System.String path) [0x00000] in <filename unknown>:0 
  at System.IO.DirectoryInfo.Create () [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectoriesInternal (System.String path) [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectory (System.String path) [0x00000] in <filename unknown>:0 

  at System.IO.DirectoryInfo.Create () [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectoriesInternal (System.String path) [0x00000] in <filename unknown>:0 
  at System.IO.Directory.CreateDirectory (System.String path) [0x00000] in <filename unknown>:0 
  at Mono.Security.X509.X509Store.CheckStore (System.String path, Boolean throwException) [0x00000] in <filename unknown>:0 
  at Mono.Security.X509.X509Store.Import (Mono.Security.X509.X509Certificate certificate) [0x00000] in <filename unknown>:0 
  at System.Security.Cryptography.X509Certificates.X509Store.Add (System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) [0x00000] in <filename unknown>:0 
  at Firebase.Unity.InstallRootCerts.Install (Firebase.FirebaseApp app) [0x00000] in <filename unknown>:0 
  at Firebase.FirebaseApp.CheckDependencies () [0x00000] in <filename unknown>:0 
  at Firebase.FirebaseApp.CreateAndTrack (Firebase.CreateDelegate createDelegate) [0x00000] in <filename unknown>:0 
  at Firebase.FirebaseApp.Create () [0x00000] in <filename unknown>:0 
  at Firebase.FirebaseApp.get_DefaultInstance () [0x00000] in <filename unknown>:0 
  at Firebase.Analytics.FirebaseAnalytics..cctor () [0x00000] in <filename unknown>:0 
  at FirebaseUnity.InitializeFirebase () [0x00000] in <filename unknown>:0 
  at FirebaseUnity.Start () [0x00000] in <filename unknown>:0
```

Please help anyone knows about it

**Comment by hue1992** [comment]
I think you remove Google Mobile Ads with Package Uninstaller in Assetstore

**Comment by PAHeartBeat** [comment]
I think it's not proper Solution Cos we also want AdMob Ads, but We already removed all(except AdMob Dependancy) play sevice resolver files came with AdMob package from Google GitHub

**Comment by BaltaKid** [comment]
Hello @PAHeartBeat,

I use firebase analytics/ message. and google mobile ads. 
The only think that i have you have to do is delete the Google.VersionHandler of Google Mobile Ads and it should work.

And check if cocoapods is updated.

Best Regards and sorry for bad english

**Comment by stewartmiles** [comment]
We discovered this issue was associated with using the Mono network stack prior to initializing Firebase would cause our certification installation to fail.  This is fixed in 4.0.3.  For more details of the release see https://firebase.google.com/support/release-notes/unity

Cheers,
Stewart
